main.go

```diff
@@ -38,12 +39,31 @@ var (
 )
 
 type Job struct {
-	JobName        string   `yaml:"name"`
-	Image          string   `yaml:"image"`
-	Namespace      string   `yaml:"namespace,omitempty"`
-	ServiceAccount string   `yaml:"service_account,omitempty"`
-	Command        []string `yaml:"command,omitempty"`
-	Args           []string `yaml:"args,omitempty"`
+	JobName        string            `yaml:"name"`
+	Image          string            `yaml:"image"`
+	Namespace      string            `yaml:"namespace,omitempty"`
+	ServiceAccount string            `yaml:"service_account,omitempty"`
+	Command        []string          `yaml:"command,omitempty"`
+	Args           []string          `yaml:"args,omitempty"`
+	Limits         map[string]string `yaml:"limits,omitempty"`
+	Requests       map[string]string `yaml:"requests,omitempty"`
+}
+
+func makeResourceList(inputMap map[string]string) corev1.ResourceList {
+	// Create a new ResourceList
+	resourceList := corev1.ResourceList{}
+
+	// Iterate through the input map and populate the ResourceList
+	for key, value := range inputMap {
+		quantity, err := resource.ParseQuantity(value)
+		if err != nil {
+			fmt.Printf("Error converting value for key %s: %v\n", key, err)
```

Comment:
the error handling doesn't look clean to me. In the main function log.Fatal is being used in case of parsing issues. I would vote to use the same approach here. What are your thoughts?